crates/ruff/src/rules/eradicate/detection.rs

```diff
@@ -6,7 +6,7 @@ use ruff_python_parser::parse_suite;
 
 static ALLOWLIST_REGEX: Lazy<Regex> = Lazy::new(|| {
     Regex::new(
-        r"^(?i)(?:pylint|pyright|noqa|nosec|region|endregion|type:\s*ignore|fmt:\s*(on|off)|isort:\s*(on|off|skip|skip_file|split|dont-add-imports(:\s*\[.*?])?)|mypy:|SPDX-License-Identifier:)"
+        r"^(?i)(?:pylint|pyright|noqa|nosec|region|endregion|type:\s*ignore|fmt:\s*(on|off)|isort:\s*(on|off|skip|skip_file|split|dont-add-imports(:\s*\[.*?])?)|mypy:|SPDX-License-Identifier:|(?i)TODO(\([^\)]*\)| @\w+)?:)",
```

Comment:
Can we find a way to do this generically, for all `task_tags`? Since this wouldn't extend to `FIXME`, etc. I'm wondering if we can instead split on parentheses on line 53, where we iterate over the task tags?

Reply:
We should probably also make sure to respect https://docs.astral.sh/ruff/settings/#task-tags

It may also be possible to implement this without using a regex, but could be a bit of a pain.

Reply:
We do respect task tags but it's not part of the regex, it's down on line 53.